content/en/docs/collector/builder.md

```diff
@@ -0,0 +1,107 @@
+---
+title: Builder
+weight: 25
+---
+
+The OpenTelemetry Collector Builder (ocb) is a command-line interface tool that
+generates sources and binaries for OpenTelemetry Collector
+[distributions](../distributions).
+
+This tool is currently being used to build both the core and the contrib
+distributions, and can be used to build custom distributions as well, allowing
+users to cherry-pick which components they need for their production usage,
+potentially even including their own custom components.
+
+## Installing
+
+You can download `ocb` from the OpenTelemetry Collector's
+[release page](https://github.com/open-telemetry/opentelemetry-collector/releases)
+on GitHub. A container image is **NOT** available at this time. Prefer to use
+the same version number that you'll use in your manifest, as the builder is only
+guaranteed to work with the OpenTelemetry Collector API matching the builder's
+version.
```

Comment:
```suggestion
Download `ocb` from the Collector's
[release page](https://github.com/open-telemetry/opentelemetry-collector/releases). 
A container image is **NOT** available at this time. We recommend that you download and use
the same version that you'll use in your manifest, since the builder is only
guaranteed to work with the OpenTelemetry Collector API matching the builder's
version.
```